mkdocs.yml

```diff
@@ -51,25 +51,33 @@ markdown_extensions:
 nav:
   - Home: index.md
   - Getting Started:
-    - Installation: Install.md
-    - Quickstart Tutorial: Demo.md
-    - Configuration: configuration2.md
+    - Overview: GettingStarted/Overview.md
+    - Installation: GettingStarted/Installation.md
+    - Configuration:
+      - Pelorus Core: GettingStarted/configuration/PelorusCore.md
+      - Pelorus Exporters: GettingStarted/configuration/PelorusExporters.md
+      - Production best practices: GettingStarted/configuration/ProductionBestPractice.md
+      - NooBaa for Long Term Storage: GettingStarted/configuration/Noobaa.md
+    - Quickstart Tutorial: GettingStarted/QuickstartTutorial.md
+    - Examples: GettingStarted/Examples.md
   - Our Philosophy:
-    - Overview: philosophy/index.md
-    - The Problem Today: philosophy/problem.md
-    - Finding the Right Bridge Outcomes: philosophy/bridge_outcomes.md
+    - Overview: philosophy/Overview.md
+    - The Problem Today: philosophy/TheProblemToday.md
+    - Finding the Right Bridge Outcomes: philosophy/BridgeOutcomes.md
     - Outcomes:
-      - Overview: philosophy/outcomes/index.md
+      - Overview: philosophy/outcomes/Overview.md
       - Software Delivery Performance: philosophy/outcomes/SoftwareDeliveryPerformance.md
-    - philosophy/pelorus.md
-  - Configuration:
-    - Architecture: Architecture.md
-    - Configuration: Configuration.md
-    - NooBaa for Long Term Storage: Noobaa.md
+    - How Pelorus Can Help: philosophy/HowPelorusCanHelp.md
+  - Architecture: Architecture.md
   - Contribution:
     - Dev Guide: Development.md
     - Upstream Support: UpstreamSupport.md
 plugins:
   - social
+  - redirects:
+      redirect_maps:
```

Comment:
what this do?

Reply:
In our Pelorus Operator documentation in a bundle we have links to:

```
Pelorus Configuration Docs
https://pelorus.readthedocs.io/en/latest/Configuration/
```

```
https://pelorus.readthedocs.io/en/latest/Install/#configure-prometheus-retention
```

And few others. So to keep those links working we need to redirect to the new locations.

Reply:
Oh now I understood. Agree with the workaround to do not touch operator, but I would open an issue to remove this when we update the operator

Reply:
Created new issue to track this.